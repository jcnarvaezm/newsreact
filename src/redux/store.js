import { createStore } from 'redux';
import infoForm from './reducers/infoForm';

const store = createStore(infoForm);
export default store;
