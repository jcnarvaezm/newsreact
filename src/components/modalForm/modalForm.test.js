import React from 'react';
import { configure, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ModalForm, { mapStateToProps } from './modalForm';
import store from '../../redux/store';
import { Provider } from 'react-redux';

configure({
  adapter: new Adapter(),
});

describe('Testing App Component', () => {
  test('testing App', () => {
    const appState = {
      firstname: '',
      lastname: '',
      phonenumber: '',
      email: '',
      emailtext: '',
      chksendme: false,
      show: false,
      showToast: false,
    };
    const ownProps = {
      firstname: 'firstname',
      lastname: 'lastname',
      email: 'email@email.com',
      phonenumber: '1234567890',
      emailtext: 'email text test',
      chksendme: true,
      show: true,
      showToast: false,
    };
    const componentState = mapStateToProps(appState, ownProps);
    const wrapper = render(
      <Provider store={store}>
        <ModalForm />
      </Provider>
    );
    expect(componentState.infoForm).toEqual(appState);
    //console.log(wrapper.html('section'));
    //expect(wrapper.html('section').length).toBeGreaterThanOrEqual(1);
  });
});
