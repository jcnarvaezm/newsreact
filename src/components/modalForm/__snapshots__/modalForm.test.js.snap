// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Form Container should not regress 1`] = `
<section
  className="container"
/>
`;
